src/v2/index.ejs

```diff
@@ -42,7 +42,7 @@
   <div id="react-root"><%%- content.body %></div>
 
   <!-- String concatenation is needed allow outputting a template tag -->
-  <%- htmlWebpackPlugin.tags.bodyTags.map((originalTag) => { const tag = { ...originalTag, attributes: { ...originalTag.attributes } }; tag.attributes['src'] = "<" + "%= cdnUrl %" + ">" + originalTag.attributes['src']; tag.attributes['async'] = true; return tag; }) %>
+  <%- htmlWebpackPlugin.tags.headTags.map((originalTag) => { const tag = { ...originalTag, attributes: { ...originalTag.attributes } }; tag.attributes['src'] = "<" + "%= cdnUrl %" + ">" + originalTag.attributes['src']; return tag; }) %>
```

Comment:
Could this `headTags` switch potentially output head-only tags here (vs body-safe script tags). I guess I'm just wondering why the switch 😄)

Reply:
The v1.x and v2.x plugins are entirely different. 🤷 

They moved where the tags are inserted internally, I'm not entirely sure why however there are some newly added compiler hooks that could possibly simplify this logic. The way we inject the CDN being the only unresolved issue.

Reply:
They also added `defer` by default so I dropped `async`.